src/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 ##################################################
 # Build a custom protoc plugin
 gz_add_executable(gz_msgs_gen Generator.cc generator_main.cc)
+find_package(Protobuf REQUIRED)
```

Comment:
I don't think that this should be needed, as the top-level `CMakeLists` file should already be finding protobuf, correct?

Reply:
That what I would have thought, but it failed build on my system. I didn't find the call to `add_subdirectory(src)` so I'm a bit lost how the CMakeLists is working in the first place.